Move changelog to a separate file to allow customization

simaishi opened the pull request:
Use with configuration override option (`-v <dir>:/root/OPTIONS`)

To use custom changelog, specify name of the changelog file in "rpm.changelog" in options.yml, and place the file along with options.yml.

Fryguy commented:
@simaishi I've been talking with @chessbyte and we are considering dropping changelog altogether in favor of a release notes document via manageiq-documentation.

simaishi commented:
@Fryguy this is for rpm changelog (what shows up when you run `rpm -q --info <package>`), but it usually contains application changelog (not just rpm spec changelog). Regardless of what should be included, this should be moved out as it includes version number, email address, etc.

As for the contents, I'm not sure what we should include:
- release name only
```
* Tue Jun 2 2020 ManageIQ <contact@manageiq.org> - 10.1.0-0.3.beta1
- Jansa Beta build
```
- Add a link to changelog or release notes, something like:
```
* Tue Jun 2 2020 ManageIQ <contact@manageiq.org> - 10.1.0-0.3.beta1
- Jansa Beta build, refer to https://xxxxx/xxxx for release notes
```
- Add a few important ones
```
* Tue Jun 2 2020 ManageIQ <contact@manageiq.org> - 10.1.0-0.3.beta1
- Jansa Beta build
- Updated Rails to 5.2
- Added OpenID-Connect or OIDC Authentication support
```

Fedora Packaging Guideline is documented here https://docs.fedoraproject.org/en-US/packaging-guidelines/#changelogs

simaishi commented:
Jansa backport details:

```
$ git log -1
commit 02ed4bf3051f5596e1384077e0825c2a7f67ac4d
Author: Jason Frey <fryguy9@gmail.com>
Date:   Thu Jun 11 14:28:25 2020 -0400

    Merge pull request #61 from simaishi/move_changelog

    Move changelog to a separate file to allow customization

    (cherry picked from commit 4a56b5347e3dea22ca657487cdbeaf0db0c578a2)
```
